tests/selfapps_gribapi_hook_probe.ps1

```
@@ -0,0 +1,267 @@
+            $epsi = New-Object System.Diagnostics.ProcessStartInfo
+            $epsi.FileName = $exePath
+            $epsi.UseShellExecute = $false
+            $epsi.RedirectStandardOutput = $true
+            $epsi.RedirectStandardError = $true
+            $eproc = [System.Diagnostics.Process]::Start($epsi)
+            $eOutTask = $eproc.StandardOutput.ReadToEndAsync()
+            $eErrTask = $eproc.StandardError.ReadToEndAsync()
+            if ($eproc.WaitForExit(30000)) {
+                $exeExit = $eproc.ExitCode
+            } else {
+                try { & taskkill.exe /F /T /PID $eproc.Id 2>$null 1>$null } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)